common/build/eslint-config-fluid/minimal.js

```diff
@@ -374,6 +375,7 @@ module.exports = {
 				"@typescript-eslint/no-invalid-this": "off",
 				"@typescript-eslint/unbound-method": "off", // This rule has false positives in many of our test projects.
 				"import/no-nodejs-modules": "off", // Node libraries are OK for test files.
+        "import/no-deprecated": "off", // Deprecated APIs are OK to use in test files.
```

Comment:
Do we want deprecated stuff in our tests? I would argue for having the same rules between test and non-test code as much as possible.

(Also indentation is off on this line)